packages/types/src/types/models-extra/dashboard-item/charts/bar.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import { CssColor } from '../../../css';
+import {
+  BaseChartConfig,
+  CartesianChartConfig,
+  CartesianChartPresentationOptions,
+  ChartType,
+  ReferenceLinesConfig,
+} from './common';
+
+export type BarChartPresentationOptions = CartesianChartPresentationOptions & {
+  color?: CssColor;
+  /**
+   * @description This can be anything from the [numeraljs library]{@link http://numeraljs.com/#format}
+   */
+  valueFormat?: string;
+  referenceLines?: {
+    targetLine?: ReferenceLinesConfig;
+  };
+};
```

Comment:
Hey Alex, I’m just resolving a few conflicts with #5399. Do you know if we support(ed) reference lines on non-bar Cartesian charts?

I can see that the three visualisations that have reference lines are all bar charts, but based on the old code, I can’t see why something like a line chart couldn’t also have a reference line 🧐

While I’m here: This has made debugging so much easier!

Reply:
Promoting `referenceLines` to `CartesianChartPresentationOptions` seems to cause some typing issues. If you reckon we should support non-bar charts, then would you be able to pair on resolving these issues? No hurry, of course

Reply:
This is a good question - I'm guessing we should support it, happy to pair at some stage 🙏

Reply:
I’ll sort out my typing issues first, and let you know if I still have trouble with moving `referenceLines`. I suspect it’ll be fine 🤞🙏